lua/toggleterm/ui.lua

```diff
@@ -316,7 +317,14 @@ function M.open_split(size, term)
     local split_win = windows[#windows]
     if config.persist_size then M.save_window_size(term.direction, split_win.window) end
     api.nvim_set_current_win(split_win.window)
-    vim.cmd(commands.existing)
+    local window_width = vim.o.columns
+    local horizontal_breakpoint =
+      require("toggleterm.config").get("responsiveness_settings").horizontal_breakpoint
```

Comment:
the config module is already imported above you only need to do `config.responsiveness_settings`